.github/workflows/performance-testing.yml

```
@@ -0,0 +1,297 @@
+              // Fallback to production URL
+              const productionUrl = 'https://alocubanoboulderfest.vercel.app';
+              console.log('Using production URL:', productionUrl);
+              core.setOutput('deployment_url', productionUrl);
+            }
+
+      - name: Wait for deployment readiness
+        env:
+          DEPLOYMENT_URL: ${{ steps.vercel-deployment.outputs.deployment_url }}
+        run: |
+          echo "⏳ Waiting for deployment to be ready: $DEPLOYMENT_URL"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)